### DISCUSSION TITLE
7.4 exercise 20 doesn't have an answer

### DISCUSSION THREAD

**Discussion by levdopa** [spamegg1/Epp-Discrete-Math-5th-solutions#14] [General]
f(n) = 2n
f(n) = 3n
works

**Comment by spamegg1** [comment]
Thanks, I'll update it!

**Reply by spamegg1** [reply]
Should be fixed [here](https://github.com/spamegg1/Epp-Discrete-Math-5th-solutions/commit/3e4212e1a9f379e0505517d92ef6b81db27da459)
